test/unit/gateways/mercado_pago_test.rb

```diff
@@ -252,6 +252,17 @@ def test_successful_verify
     assert response.test?
   end
 
+  def test_successful_verify_with_custom_amount
+    response = stub_comms do
+      @gateway.verify(@credit_card, @options.merge({ amount: 200}))
```

Comment:
Rubocop requests a space after the closing bracket like so: `({ amount: 200 }))`